src/simulation/m_time_steppers.fpp

```diff
@@ -258,6 +268,20 @@ contains
             @:ACC_SETUP_SFs(rhs_vf(i))
         end do
 
+        ! Allocating the cell-average RHS time-stages for adaptive RKCK stepper
+        if (lag_bubbles) then
+            @:ALLOCATE(rhs_ts_adapt(1:6))
+            do i = 1, 6
```

Comment:
`6` is a magic number here. please fix 😄

Reply:
to be fixed still

Reply:
fixed